workers/default/image/image_scaling.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+
+from PIL import Image as PilImage
+
+
+def scale_image(image_path, max_width, max_height, target_dir):
+    """
+    Scale the image to the given size.
+
+    Open the given image and scales the size to the given values for
+    width and height.
+    The new file will be named like the original one with the new size added
+    with underscores.
+
+    Tested working for JPG and TIF.
+    """
+    logging.getLogger(__name__).debug(f"Resizing {image_path} "
+                                      f"to size: {(max_width, max_height)}")
+    file_name = os.path.splitext(os.path.basename(image_path))[0]
+    file_extension = os.path.splitext(os.path.basename(image_path))[1]
+    new_file_name = file_name +\
+        "_" + str(max_width) +\
+        "_" + str(max_height) +\
+        "." + file_extension
+
+    # conversion is needed for tiffs
+    image = PilImage.open(image_path).convert('RGB')
```

Comment:
Kann man ein tif wirklich nicht anders rescalen? (Außerdem hatten wir uns glaube ich auf die Schreibweise tif geeinigt, in comments aber vielleicht auch egal)

Reply:
Und vielleicht dann auch nur für Tifs converten? Und für jpgs nicht?

Reply:
Afaik hatten uns darauf geeinigt die Kurzform TIF und JPG zu benutzen, wenn Endungen generiert werden und Ähnliches. Aber der Name der Formate ist nunmal JPEG und TIFF, also würde ich die in der Doku auch so nennen.

Reply:
Ehrlich gesagt weiss ich nicht genau, was die Auswirkungen der Konversion nach RGB sind. 
Habe aber festgestellt, dass er bei TIFFs abschmiert, wenn man das nicht macht. Und da ich bei JPEG keinen Unterschied feststellen konnte, mache ich es einfach für alles (obwohl für JPEG nicht nötig).

Reply:
Der Schritt sollte für JPG tatsächlich keinen Unterschied machen weil RGB dort der Standardfarbraum zu sein scheint.